packages/capacitor-plugin/android/src/main/kotlin/com/outsystems/capacitor/plugins/geolocation/OSGeolocationErrors.kt

```diff
@@ -0,0 +1,76 @@
+package com.outsystems.capacitor.plugins.geolocation
+
+/**
+ * Object with plugin errors
+ */
+object OSGeolocationErrors {
```

Comment:
Maybe we could remove the`OS` here and keep the same naming pattern as `GeolocationPlugin`.